### ISSUE TITLE
MQTT

### ISSUE THREAD

**Issue by wellknownasmax** [ecocurious/MultiGeiger#462]
Ist es geplant MQTT zu integrieren? Ich würde die Messwerte in meinem ioBroker darstellen und weiß nicht wie ich das sonst lösen sollte.

**Comment by ituri** [comment]
Gibt es schon eine Abschätzung, wann die neue Version verfügbar sein wird?

**Comment by PapaErde** [comment]
Ich erlaube mir obige Frage ebenso zu stellen. MQTT wär fein <3 
Hier, um den Multigeiger in den Homeassistant zu itegrieren. Dank und Gruß!

**Comment by urs8000** [comment]
ich wünsche mir, dass (endlich) analog dem "airrohr", das ja aus der gleichen ecke kommt, eine MQTT übertragung möglich wird.
damit ist die integration in Homeassistant und andere einfach möglich.
danke für die schnelle umsetzung

**Comment by pbriesch** [comment]
Ja, das wäre wirklich toll!Am 06.09.2023 um 15:54 schrieb urs ***@***.***>:﻿
ich wünsche mir, dass (endlich) analog dem "airrohr", das ja aus der gleichen ecke kommt, eine MQTT übertragung möglich wird.
damit ist die integration in Homeassistant und andere einfach möglich.
danke für die schnelle umsetzung

**Comment by PapaErde** [comment]
Nachdem leider keine Timeline für eine MQTT Implementierung da ist, habe ich unter https://www.maltepoeggel.de/?site=iotgeiger eine Alternative mit MQTT Unterstützung gefunden. Malte hatte auch eine Platine zum Verkauf, vllt gibts ja für Interessierte noch Bestände.
Multigeiger bleibt für mich das Handheld-Gerät, an dieser Stelle vielen Dank nochmal an den Entwickler. iotgeiger wird via MQTT im HomeAssistant integriert.

**Comment by urs8000** [comment]
> Nachdem leider keine Timeline für eine MQTT Implementierung da ist, habe ich unter https://www.maltepoeggel.de/?site=iotgeiger eine Alternative mit MQTT Unterstützung gefunden. Malte hatte auch eine Platine zum Verkauf, vllt gibts ja für Interessierte noch Bestände. Multigeiger bleibt für mich das Handheld-Gerät, an dieser Stelle vielen Dank nochmal an den Entwickler. iotgeiger wird via MQTT im HomeAssistant integriert.

danke für die info @PapaErde. leider geht mit dieser implementierung die datenlieferung an ecocurious flöten.
mal sehen, ob aus der ESPhome-implementierung ausser dem yaml was rauszulesen ist
